Source/WorkersES6/decodeDraco.js

```diff
@@ -275,6 +275,24 @@ function decodePointCloud(parameters) {
 function decodePrimitive(parameters) {
   var dracoDecoder = new draco.Decoder();
 
+  // Skip all parameter types except generic
+  // Note: As a temporary work-around until GetAttributeByUniqueId() works after
+  // calling SkipAttributeTransform(), we will not skip attributes with multiple
+  // sets of data in the glTF.
+  var attributesToSkip = ["POSITION", "NORMAL"];
+  var compressedAttributes = parameters.compressedAttributes;
+  if (!defined(compressedAttributes["COLOR_1"])) {
+    attributesToSkip.push("COLOR");
+  }
+  if (!defined(compressedAttributes["TEXCOORD_1"])) {
+    attributesToSkip.push("TEX_COORD");
+  }
```

Comment:
glTF does allow for `TEXCOORD_2`, `TEXCOORD_3`, etc so some of this glTF attribute name -> draco name mapping could be organized in a more general way. Though to be fair, `TEXCOORD_2` and higher are only used in `ModelExperimental` and if we're expecting a bug fix in draco soon then maybe we don't have to worry about it here.

Reply:
My thinking was that if there were more than one TEXCOORD, they numbering would go in order, meaning that at least `TEXCOORD_1` would exist - in which case, we would not call `SkipAttributeTransform()`. Is it possible to just have one TEXCOORD and call it `TEXCOORD_2`? I'm not sure I'm following your comment.

Reply:
Oh that's true, never mind then. From the glTF spec:

> All indices for indexed attribute semantics MUST start with 0 and be consecutive positive integers